host-os-iso-creator.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) IBM Corp. 2016.
```

Comment:
Ok, rename done ("-_ -> "_"), I've just used dash to follow the repository pattern.